CreeDictionary/utils/fst_analysis_parser.py

```diff
@@ -12,6 +12,38 @@
 )
 
 
+def split_analysis(analysis: str) -> Tuple[List[str], FSTLemma, List[str]]:
+    """
+    :return: the tags before the lemma, the lemma itself, the tags after the lemma
+    :raise ValueError: when the analysis is not parsable.
+
+    >>> split_analysis('PV/e+fakeword+N+I')
```

Comment:
Do these doctests run as part of our unit test suite? If not, can you make them run? http://doc.pytest.org/en/latest/doctest.html

Reply:
pretty sure it runs. We have `addopts = --doctest-modules` in `setup.cfg`.

To be sure though, I just added this function to the same file and ran the unit test. And our unit test **failed succesfully**.
```
def is_doc_utils_run():
    """
    >>> print(1)
    2
    :return:
    """
    pass
```